src/vs/editor/common/model/tokenizationTextModelPart.ts

```diff
@@ -104,35 +109,15 @@ export class TokenizationTextModelPart extends TextModelPart implements ITokeniz
 		this._semanticTokens.flush();
 	}
 
+	// TODO@hediet TODO@alexdima what is the difference between this and acceptEdit?
```

Comment:
:+1: I agree that they should be merged, right now `acceptEdit` only updates tokens and `handleDidChangeContent` only updates states. Maybe in a separate cleanup PR